contents/handbook/growth/marketing/posthog-style-guide.md

```diff
@@ -140,12 +166,11 @@ posthog.capture('user_signed_up', {
 })
 ```
 
-### Always provide a link
+### Record videos
 
-Don't just write "You can contact us to learn more" and not link it to anything.
+For flows with many steps, it's often more helpful to include a video instead of a screenshot. We use [screen studio](https://screen.studio/) to record videos.
```

Comment:
```suggestion
For flows with many steps, it's often more helpful to include a video instead of a screenshot. We use [Screen Studio](https://screen.studio/) to record videos, feel free to buy yourself.
```